pocketimdb/movies/models.py

```diff
@@ -23,3 +23,12 @@ class MovieLike(models.Model):
 
     class Meta:
         unique_together = ('movie', 'user',)
+
+class MovieComment(models.Model):
+    movie = models.ForeignKey(Movie, on_delete=models.CASCADE, related_name='movie_comments')
+    user = models.ForeignKey(User, on_delete=models.CASCADE, related_name='user_comments')
+    comment = models.CharField(max_length=500)
```

Comment:
ovo moze biti `content`, jer svkakao imas model comment koji u sebi ima razne inofmraicje, pa tako i sam `content` koji cini poruku

Reply:
Da ja sam u serializer-u bas i menjao to polje iz comment u content, da ne bih model ponovo menjao, ali izmenicu onda to i u modelu. Ima mnogo vise smisla.